### ISSUE TITLE
C++11 support: std::stol, std::stoul, std::to_string and probably some other functions from C++11 are not declared.

### ISSUE THREAD

**Issue by dbanet** [psmedley/gcc#14]
Expected behavior:

```
# uname
Linux
# cat <<EOF | g++ -std=c++11 -x c++ -
> #include <string>
> #include <iostream>
> using namespace std;
> main(){ string s="42";
> cout<<stol(s)<<endl; }
> EOF
# ./a.out
42
```

Present behavior:

```
# uname
OS/2
# cat <<EOF | g++ -std=c++11 -x c++ -
> #include <string>
> #include <iostream>
> using namespace std;
> main(){ string s="42";
> cout<<stol(s)<<endl; }
> EOF
<stdin>: In function 'int main()':
<stdin>:5:13: error: 'stol' was not declared in this scope
```

**Comment by komh** [comment]
Workaround: `g++ -std=gnu++11 -x c++ -D_GLIBCXX_USE_C99`

**Comment by komh** [comment]
I've fixed this problem in #23.

**Comment by komh** [comment]
Fixed by commit fef1630c35955e6352f9734b43d20758ce7508be.
